src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -87,12 +87,10 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output): void
     {
         if (!$this->receiverLocator->has($receiverName = $input->getArgument('receiver'))) {
-            throw new \RuntimeException(sprintf('Receiver "%s" does not exist.', $receiverName));
+            throw new RuntimeException(sprintf('Receiver "%s" does not exists.', $receiverName));
```

Comment:
That's not a typo. Receiver **does** or **doesn't** exist :)

Reply:
Reverted, thanks! :smile: